File: MinimalSquare.cpp
#include <bits/stdc++.h>
using namespace std;

int main(void) {
    // variable declaration and input
    int t;
    cin >> t;
    vector<int> ans;

    while (t--) {
        int a, b;
        cin >> a >> b;

        // both rectangles should touch one of the sides
        int side = min(max(2*a, b), max(a, 2*b)); 
        ans.push_back(side*side); // area of square
    }

    // output and returning 0
    for (auto i=ans.begin(); i!=ans.end(); i++) 
        cout << *i << endl;

    return 0;
}

File: Python/MinimalSquare.py
def main():
    t = int(input())
    for _ in range(t):
        a, b = map(int, input().split())
        mi = min(a, b)
        ma = max(a, b)
        if mi*2 < ma:
            print(ma**2)
        else:
            print((mi*2)**2)


if __name__ == '__main__':
    main()
